aws/policy/compute.yaml

```diff
@@ -114,7 +118,6 @@ Statement:
     Effect: Allow
     Action:
       - autoscaling:Describe*
-      - ec2:DescribeAvailabilityZones
```

Comment:
This one's a special case.  Most of our ec2:Describe operations are limited by region, DescribeAvailabilityZones doesn't like being restricted.